modules/dists/CyclicDist.chpl

```diff
@@ -125,15 +125,11 @@ As demonstrated by the above example, a `forall` loop over a
 ``cyclicDist``-distributed domain or array executes each iteration on
 the locale owning the index in question.
 
-By default, parallelism within each locale is applied to that locale's
+Parallelism within each locale is applied to that locale's
```

Comment:
I'd be OK with preserving "By default," here (and in the Block case) because even though we don't have a stable way of changing that default, there are ways to do it (and I think alluding to that potential even if we don't answer the obvious question here in these admittedly high-level docs is reasonable from advertising/intention purposes).